vmm/src/landlock.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright © 2024 Microsoft Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::convert::TryFrom;
+use thiserror::Error;
+
```

Comment:
Don't use whitespace in imports - because cargo fmt can't then enforce ordering correctly.

Reply:
fixed